cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraCluster.java

```diff
@@ -569,6 +569,11 @@ private CassandraNode buildCassandraNode(Protos.Offer offer, boolean seed, Strin
         CassandraNode.Builder builder = CassandraNode.newBuilder()
                 .setHostname(offer.getHostname())
                 .setTargetRunState(CassandraNode.TargetRunState.RUN)
+                .addDataVolumes(
+                    DataVolume.newBuilder()
+                        .setPath(configuration.getDefaultConfigRole().getPreDefinedDataDirectory())
+                        .build()
+                )
```

Comment:
OK - then we have the optional {{sizeMb}} field to check whether it needs to be migrated.